docs/content/_navigation.json

```diff
@@ -123,7 +123,21 @@
         "children": [
           {
             "title": "Partitions",
-            "path": "/concepts/partitions-schedules-sensors/partitions"
+            "path": "/concepts/partitions-schedules-sensors/partitions",
+            "children": [
+              {
+                "title": "Partitioned assets",
+                "path": "/concepts/partitions-schedules-sensors/partitioned-assets"
+              },
+              {
+                "title": "Partitioned ops (Advanced)",
+                "path": "/concepts/partitions-schedules-sensors/partitioned-ops"
+              },
```

Comment:
take it or leave it - should they be "partitioning assets/ops" vs "partitioned"? so 1) the naming pattern is more consistent with "testing partitions", 2) it's more about the use case / behavior of partitioning, than defining the assets or ops themselves?

Reply:
We can do that; I don't feel strongly about it